package.json

```diff
@@ -106,7 +111,7 @@
     "@cerner/terra-enzyme-intl": "^4.0.0",
     "@cerner/terra-functional-testing": "^3.0.0",
     "@cerner/terra-open-source-scripts": "^1.1.0",
-    "@cerner/webpack-config-terra": "^3.0.0",
+    "@cerner/webpack-config-terra": "file:./cerner-webpack-config-terra-1.3.1.tgz",
```

Comment:
I would suggest you to consume latest of webpack-config-terra and make required changes on the latest package 3.1.0 instead of 1.3.1.

Reply:
Ok, thx!
Will do